src/components/Persona/Persona.tsx

```diff
@@ -35,17 +36,19 @@ export class Persona extends React.Component<IPersonaProps, any> {
         ) }
 
         { presence !== PersonaPresence.none && <div className='ms-Persona-presence'></div> }
-
         { !hidePersonaDetails && (
-          <div className='ms-Persona-details'>
+          <div className= { personaDetailsClass }>
             <div className='ms-Persona-primaryText'>{ primaryText }</div>
             { secondaryText ? (
               <div className='ms-Persona-secondaryText'>{ secondaryText }</div>
             ) : (null) }
             <div className='ms-Persona-tertiaryText'>{ tertiaryText }</div>
             <div className='ms-Persona-optionalText'>{ optionalText }</div>
-          </div>) }
-      </div>
+            { innerControl ? (
```

Comment:
Talked with @abenems , instead of this we will simply render { this.props.children } and let the caller do what they need here.